lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -277,6 +277,14 @@ export class UserAgentApplication {
         }
     }
 
+    /**
+     * Adding public interface for urlContainsHash
```

Comment:
This seems like it could be the commit message, not teh comment :)

Reply:
Can use the comment from the `UrlUtils` file: "Check if the url contains a hash with known properties"

Reply:
My mistake, lazy coding. Corrected now. Please check.